content/en/blog/_posts/2024-09-30-gateway-api-v1.2/index.md

```diff
@@ -0,0 +1,418 @@
+that it thinks you're using: check out [the release notes] for more
+information about what you need to do to safely upgrade.
+
+[the release notes]: https://github.com/kubernetes-sigs/gateway-api/releases/tag/v1.2.0
+
+### Experimental `Gateway.status.supportedFeatures`
+
+A much smaller breaking change, `Gateway.status.supportedFeatures` is now a
+list of objects instead of a list of strings. The objects have a single `name`
+field, so the translation from the strings is straightforward, but moving to
+objects permits a lot more flexibility for the future. This stanza is not yet
+present in the Standard channel.
+
+## Graduations to the Standard Channel
+
+Gateway API 1.2.0 graduates four features to the Standard channel. This means
+they are no longer experimental concepts; inclusion in the Standard release
+channel denotes a high level of confidence in the API surface and provides
+guarantees of backward compatibility. Of course, as with any other Kubernetes
+API, Standard Channel features can continue to evolve with backward-compatible
+additions over time, and we certainly expect further refinements and
+improvements to these new features in the future. For more information on how
+all of this works, refer to the [Gateway API Versioning
+Policy](https://gateway-api.sigs.k8s.io/concepts/versioning/).
+
+#### [HTTPRoute Timeouts](https://gateway-api.sigs.k8s.io/guides/http-routing/)
+
+[GEP-1742](https://gateway-api.sigs.k8s.io/geps/gep-1742/) introduced the
+`timeouts` stanza into HTTPRoute, permitting configuring basic timeouts for
+HTTP traffic. This is a simple but important feature for proper resilience
+when handling HTTP traffic, and it is now Standard.
+
+For example, this HTTPRoute configuration sets a timeout of 300ms for traffic
+to the `/face` path:
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: face-with-timeouts
+  namespace: faces
+spec:
+  parentRefs:
+    - name: my-gateway
+      kind: Gateway
+      port: 80
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /face
+    backendRefs:
+    - name: face
+      port: 80
+    timeouts:
+      request: 300ms
+```
+
+For more information, check out the [HTTP routing
+documentation](https://gateway-api.sigs.k8s.io/guides/http-routing/). (Note
+that this applies only to HTTPRoute timeouts. GRPCRoute timeouts are not yet
+part of Gateway API.)
+
+#### [Gateway Infrastructure Labels and Annotations](https://gateway-api.sigs.k8s.io/guides/grpc-routing/)
+
+Gateway API for north-south traffic has always had the concept that declaring
+a Gateway causes the backing infrastructure (for example, a Service and a
+Deployment) to be created by the Gateway API controller. One complication with
+this approach is that sometimes it's important to label or annotate the
+infrastructure: for example, it's commonly necessary to use one of these
+mechanisms to tell a service mesh to incorporate the Gateway infrastructure.
```

Comment:
Maybe something like this to remove the north-south references and expand this to include implementations that generate non-Kube resources:
```suggestion
In many cases, Gateway API implementations are creating additional resources to make each Gateway work. For example, implementations that run in-cluster often create Kubernetes Services and Deployments, while cloud-based implementations may be creating Cloud LB resources. It can be helpful to have some common labels or annotations propagated to those generated resources.
```